### PR TITLE
docs: ensure that optionality matches between documented params and signatures

### PR DESCRIPTION
**Pull request by MarshallOfSound**
This pulls in a new docs parser version that includes some enforcements to ensure that the optionality we define with `(optional)` also matches the optionality in the signature indicated with `(a[, optionalB])`.

Using that enforcement I then fixed a bunch of issues 👍 

Notes: no-notes

### CODE REVIEW

File: docs/api/browser-window.md [hunk 1]
```diff
@@ -864,7 +864,7 @@ Returns `Boolean` - Whether the window is in normal state (not maximized, not mi
 
 * `aspectRatio` Float - The aspect ratio to maintain for some portion of the
 content view.
-* `extraSize` [Size](structures/size.md) - The extra size not to be included while
+* `extraSize` [Size](structures/size.md) (Optional) - The extra size not to be included while
```
Comment (nornagon) [inline docs/api/browser-window.md]: any particular reason not to be consistent re: capitalization between `(Optional)` and `(optional)`?

Comment (MarshallOfSound) [reply docs/api/browser-window.md]: Fair, enforcing lower case now 👍

**Comment by codebytere** [comment]
@MarshallOfSound does this still need to get backported?

**Comment by MarshallOfSound** [comment]
@codebytere Entirely optional 👍

**Pull request merged**